docker/Dockerfile.hpx-env

```diff
@@ -1,7 +1,7 @@
 FROM quay.io/pypa/manylinux_2_28_x86_64:2022-11-19-1b19e81
 
 RUN dnf update -y
-RUN dnf install -y ninja-build hwloc-devel
+RUN dnf install -y ninja-build hwloc-devel graphviz graphviz-devel
```

Comment:
This is uneeded. This is used to extend the main docker file.

Reply:
It could be separated in the future maybe, it doesn't hurt to keep it in both places IMO

Reply:
It will hurt any time we modify it and it adds confusion.
See: https://github.com/zama-ai/concrete/pull/668/files#diff-752b1613bc11ef1a7b8a75a0003826ba4c1abec9901197206e127ce2756bf650R24

Reply:
As you wish.